### PR TITLE
[CLNP-3674] feat: RTL(Right To Left) display support

### PR DESCRIPTION
**Pull request by AhyoungRyu**
Addresses https://sendbird.atlassian.net/browse/CLNP-3543 

I added `htmlTextDirection` prop(The original element attribute name is `dir` but I chose to be more elaborate in our app) to SendbirdProvider to support Right To Left(RTL) text direction display for the middle east customers. 
https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dir

### CODE REVIEW

File: src/stories/apps/GroupChannelApp.stories.tsx
Comment (AhyoungRyu) [inline src/stories/apps/GroupChannelApp.stories.tsx:140]: Screenshot 2024-06-11 at 4 06 45 PM
Added this one too for the easier test :)

**Comment by bang9** [comment]
Could you generate a preview URL for testing?

**Review by bang9 — COMMENTED** [review]
~~Shouldn't we lower it to the module level instead of the app level?~~
~~And when customers customize and reuse components, we might need to lower the level even further for it to apply.~~

checked docs: https://sendbird.atlassian.net/wiki/spaces/SDK/pages/2401009678/RTL+Support

**Comment by AhyoungRyu** [comment]
@bang9 Sure. Here the link -> https://deploy-preview-1132--sendbird-uikit-react.netlify.app/?htmlTextDirection=rtl

**Comment by bang9** [comment]
One consideration we need to address is whether it is acceptable for this to be merged into the `main` branch without the layout being fixed. As you know, deployments can happen at any time on the `main` branch, and if that happens, this feature might be exposed to customers in an incomplete state.

**Review by bang9 — APPROVED** [review]

**Comment by AhyoungRyu** [comment]
@bang9 Oh I thought there's no one who uses this until we make this public 😄 Then I won't merge this one and keep working based on this branch. 

@HoonBaek Please create a PR based this one 🙌

**Review by chrisallo — APPROVED** [review]
lgtm!

**Comment by AhyoungRyu** [comment]
I think it's ready so I'm gonna deliver the preview link to the design & QA team. But I'll merge this on next Tuesday(18th) so it can be included in the next regular release.

**Comment by AhyoungRyu** [comment]
I got an approval from QA team so it seems okay to merge this into main branch 🎉 
https://sendbird.slack.com/archives/C0794CS06EM/p1719386625273959 

FYI, I got separate approvals for sub-PRs I made after I got 🆗 from Airen & Chris :)

**Pull request merged**
